src/services/api/account.js

```diff
@@ -188,11 +189,86 @@ export const createAccountBrainkey = async ({ name, brainkey, email }) => {
   return createAccount({ name, activeKey, ownerKey, email });
 };
 
+function createWalletBackup(
+  passwordPubkey,
+  walletObject,
+  compressionMode,
+  entropy
+) {
+  return new Promise(resolve => {
+    const publicKey = PublicKey.fromPublicKeyString(passwordPubkey);
+    const onetimePrivateKey = key.get_random_key(entropy);
+    const walletString = JSON.stringify(walletObject, null, 0);
+    lib.LZMA_WORKER.compress(walletString, compressionMode, compressedWalletBytes => {
+      const backupBuffer = Aes.encrypt_with_checksum(
+        onetimePrivateKey,
+        publicKey,
+        null /* nonce */,
+        compressedWalletBytes
+      );
+
+      const onetimePublicKey = onetimePrivateKey.toPublicKey();
+      const backup = Buffer.concat([
+        onetimePublicKey.toBuffer(),
+        backupBuffer
+      ]);
+      resolve(backup);
+    });
+  });
+}
+
+// comments get from bitshares-ui
+export const generateBackupFileFromBrainkey = async ({ brainkey, password }) => {
+  const accountId = await getAccountIdByBrainkey(brainkey);
+  const { name } = await getUser(accountId);
+  const passwordPrivate = PrivateKey.fromSeed(password);
```

Comment:
Why do you need a password? Assume the wallet is unlocked

Reply:
I need password as field for encrypt in backup file for bitshares-ui users